pkg/messaging/direct_messaging.go

```diff
@@ -237,7 +237,7 @@ func (d *directMessaging) setContextSpan(ctx context.Context) context.Context {
 }
 
 func (d *directMessaging) invokeRemote(ctx context.Context, appID, namespace, appAddress string, req *invokev1.InvokeMethodRequest) (*invokev1.InvokeMethodResponse, error) {
-	conn, teardown, err := d.connectionCreatorFn(context.TODO(), appAddress, appID, namespace, false, false, false)
```

Comment:
I don't think we should change it here, it was meant as a TODO on purpse IMHO

Reply:
Ok, i change it back.

Reply:
ok . i change it back.